.github/workflows/e2e-nvidia-a10g-x8.yml

```diff
@@ -1,6 +1,6 @@
 # SPDX-License-Identifier: Apache-2.0
 
-name: E2E (NVIDIA A10G x4)
+name: E2E (NVIDIA A10G x8)
```

Comment:
The 8xA10 nodes are $16 / hour, have poor availability and can't run mixtral + lora adapters (when we get to that point)

The 4xL40s nodes g6e.12xlarge are $10/hour and can (48gb vram per card) . They also support a newer instruction set. Can we switch to those?

Reply:
Yup, that sounds good to me!

Reply:
+1